config/environments/production.rb

```diff
@@ -58,7 +58,7 @@
   config.log_tags = [:request_id]
 
   # Use a different cache store in production.
-  # config.cache_store = :mem_cache_store
+  config.cache_store = :redis_cache_store, { url: "#{ENV['REDIS_URL']}/1" }
```

Comment:
```suggestion
  config.cache_store = :redis_cache_store, { url: "#{ENV.fetch('REDIS_URL')}/1" }
```

This should fail to boot if no REDIS_URL.